spec/scopes-and-launch-context/index.md

```diff
@@ -273,6 +272,13 @@ Scope              | Grants
 `online_access`    | Request a `refresh_token` that can be used to obtain a new access token to replace an expired one, and that will be usable for as long as the end-user remains online.
 `offline_access`   | Request a `refresh_token` that can be used to obtain a new access token to replace an expired token, and that will remain usable for as long as the authorization server and end-user will allow, regardless of whether the end-user is online.
 
+## Extensions
+
+Additional context parameters and scopes can be used as extensions using the following namespace conventions:
+
+- use a *full URI* that you control (e.g. http://example.com/scope-name)
+- use any string starting with "__"
```

Comment:
Can you quote this as code and add an explanatoin, so it renders like:

- use any string starting with `__` (two underscores)